Makefile

```diff
@@ -23,6 +23,8 @@
 
 SRCS := $(wildcard src/*.cpp)
 TESTS := $(wildcard test/*.cpp)
+DOCUMENTATION := $(wildcard documentation/*.man)
+DOCUMENTATION_FILES := $(patsubst documentation/man-%.h, documentation/%.man,$(DOCUMENTATION))
```

Comment:
Had to switch the first and third arguments to patsubst but I think it should work as expected now. 

I assume you mean remove the man-%.h files? since those are the ones generated, not the .man

Reply:
yep, you're right!